tools/releaseBuild/azureDevOps/releaseBuild.yml

```diff
@@ -215,6 +215,49 @@ stages:
             - sign_windows_fxdependent_release
             - sign_windows_fxdependentWinDesktop_release
 
+- stage: PublishSymbol
+  displayName: Publish symbols
+  dependsOn: Build
+  jobs:
+  - job: Symbols_Job
+    pool:
+      name: PowerShell1ES
+      demands:
+      - ImageOverride -equals PSMMS2019-Secure
+    steps:
+    - download: current
+      artifact: results
+      patterns: '**/*.zip'
+
+    - pwsh: |
+        Write-Verbose -Verbose "Enumerating $(Pipeline.Workspace)/results"
+        $downloadedArtifacts = Get-ChildItem -Recurse "$(Pipeline.Workspace)/results"
+        $downloadedArtifacts
+        $expandedRoot = New-Item -Path "$(Pipeline.Workspace)/expanded" -ItemType Directory -Verbose
+        $symbolsRoot = New-Item -Path "$(Pipeline.Workspace)/symbols" -ItemType Directory -Verbose
+        $downloadedArtifacts | Where-Object { $_.Name -like 'PowerShell*-symbols*-signed.zip'} | ForEach-Object {
```

Comment:
The signed file doesn't contain the symbols..  use the unsigned.  More details in the hidden comment (my notes investigated the issue).

The symbols are in a symbols.zip inside that file.